packages/ui/src/form.tsx

```diff
@@ -146,7 +146,9 @@ const FormMessage = React.forwardRef<
 			body = String(error.message);
 		}
 		if (Array.isArray(error)) {
-			body = error[0]?.message;
+			const firstErrorIndex = error.findIndex((e) => !!e);
+			const firstError = error[firstErrorIndex];
+			body = firstError?.message ?? `Error with value at index ${firstErrorIndex}`;
```

Comment:
this isn't a great error message, but I wasn't sure a better way to get something to show up when there is an error with a value in one of the array fields. This shows up as 
![image](https://github.com/user-attachments/assets/e968b276-7a4b-4afa-b867-5e825b896875)
 when the 'value' field is empty